admin/pages/dashboard.php

```diff
@@ -230,6 +229,7 @@
 		</div>
 	</div>
 	<div id="webmaster-tools" class="wpseotab">
+		<h3><?php _e( 'Verification Webmaster Tools', 'wordpress-seo' ); ?></h3>
```

Comment:
Perhaps 'Webmaster Tools Verification' is better instead of 'Verification Webmaster Tools'